### ISSUE TITLE
Creating a validation dataset

### ISSUE THREAD

**Issue by Samiiha28** [mwanjajoel/clothia#1]
When splitting the dataset it is always advisable to split it into train, validation, and test set. That is if you are not using **cross-validation.**  I do not want to go into the details of cross-validation since it's your first project but you will meet it along the way.
Your dataset is very huge (60k images). You can create a small validation set  (it could be about >=5k images). This one helps you monitor the performance of the model during training i.e if the model is overfitting or underfitting. This information will let you know when to halt the training process. 

More information about the validation set can be found here https://machinelearningmastery.com/difference-test-validation-datasets/
https://www.kdnuggets.com/2017/11/create-good-validation-set.html
